src/test/java/it/slawekpaciorek/services/InMemoryDBServiceTest.java

```diff
@@ -0,0 +1,224 @@
+package it.slawekpaciorek.services;
+
+import it.slawekpaciorek.model.Product;
+import it.slawekpaciorek.model.UserOrder;
+import it.slawekpaciorek.repo.InMemoryDB;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.assertEquals;
+
+public class InMemoryDBServiceTest {
+
+    InMemoryDBService SUT = new InMemoryDBService();
+    List<UserOrder> orders;
+
+    @Before
+    public void setOrders(){
+        orders = new ArrayList<>();
+        Product product1 = new Product("Bułka", 1, 10.0);
+        Product product2 = new Product("Chleb", 2, 30.0);
+        Product product3 = new Product("Cola", 3, 34.60);
+        List<Product> products = new ArrayList<>();
+        products.add(product1);
+        products.add(product2);
+        products.add(product3);
+        UserOrder order1 = new UserOrder(1,1, products);
+        UserOrder order2 = new UserOrder(2,1, products);
+        UserOrder order3 = new UserOrder(3,1, products);
+        UserOrder order4 = new UserOrder(1,2, products);
+        UserOrder order5 = new UserOrder(2,2, products);
+        orders.add(order1);
+        orders.add(order2);
+        orders.add(order3);
+        orders.add(order4);
+        orders.add(order5);
+    }
+
+    @After
+    public void cleanDB(){
+        InMemoryDB.cleanUp();
+    }
+
+    @Test
+    public void shouldCheckForOrders(){
+
+//        Given
+        List<UserOrder> expected = new ArrayList<>();
+//        When
+        List<UserOrder> result = SUT.findAllOrders();
+//        Then
+        assertEquals(expected, result);
+
+    }
+
+    @Test
+    public void shoudCheckOrdersForUser(){
+//        given
+        long id = 1;
+        List<UserOrder> expected = orders.stream().filter(x->x.getUserId()==id).collect(Collectors.toList());
```

Comment:
Does this actually test anything:
    public List<UserOrder> findOrderForUser(long userId){
       ....
        return InMemoryDB.getOrders().stream().filter(x -> x.getUserId() == userId).collect(Collectors.toList());
    }
You are actually executing same method just in two different places and check if resutls are the same